recipes/longhorn.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: kbrew
+app:
+  repository:
+    name: longhorn
+    url: https://charts.longhorn.io
+    type: helm
+  namespace: "longhorn-system"
+  pre_install:
+  - steps:
+    - curl -s https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/scripts/environment_check.sh | bash
+  post_install:
+  - steps:
+    - echo ${USER}:$(echo ${PASSWORD} | openssl passwd -stdin -apr1) >> auth
```

Comment:
I think we have to somehow call out openssl dependency - may break on some instances!
That makes me think should we be running kbrew inside a container - as we have some dependencies such as jq, openssl etc.?

Reply:
@vishal-biyani running inside the container will make it complicated managing global settings. We will also have to mount config files/volumes

Reply:
We can run the 3rd party commands like OpenSSL and jq inside containers

Reply:
@sahil-lakhwani as discussed, let's add a prerequisites check in the recipes for now for now and fail early if the required CLIs are not installed

Reply:
@mahendrabagul could you please check if there is another way to achieve this?

Reply:
I've added a check for now to verify that `openssl` is present